android/app/src/main/res/values/strings.xml

```diff
@@ -1,9 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <resources>
-    <!-- Shown on app info, app switcher, etc -->
-    <string name="app_name" translatable="false">PathCheck BT</string>
-    <string name="exposure_notification_channel_description">Exposure notification alerts.</string>
-    <string name="exposure_notification_channel_name">Exposure Notification</string>
-    <string name="exposure_notification_message">Someone you were near has tested positive for COVID-19. Tap for more info.</string>
-    <string name="exposure_notification_title">Possible COVID-19 exposure</string>
+  <!-- Shown on app info, app switcher, etc -->
+  <string name="app_name" translatable="false">PathCheck BT</string>
+  <!-- Shown below icon on launcher only -->
+  <string name="app_name_short" translatable="false">PathCheck BT</string>
```

Comment:
We don't need this string anymore, we could remove it from Lokalise the next time we pull strings